templates/app/views/checkouts/steps/_payment_step.html.erb

```diff
@@ -3,60 +3,63 @@
     <legend>
       <%= t('spree.payment_information') %>
     </legend>
+    <% if @order.covered_by_store_credit? %>
+      <p>Your order is fully covered by store credits, no additional payment method is required.</p>
```

Comment:
This message needs to be associated with a translation key.

Reply:
https://github.com/solidusio/solidus/pull/5208

Reply:
as pointed out by @kennyadsl in solidusio/solidus#5208 we can start storing frontend related keys in the starter itself, with time we should end up with a tighter, more coherent list of translations for the core instead of the giant ball of unused ones we currently have.

Reply:
True. Still, I'd like to keep that change separate from this PR. This change will end up in the starter fronted 4.2 so I think the current approach of adding the translation key to the core is safe because Solidus Starter Frontend 4.2 will only be installed on Solidus 4.2. 

I think we can go as Rainer suggested and open an issue for moving the translation keys from the core over here (I expect the usual backward compatibility pain 😅 ).

Reply:
@kennyadsl I think there are ways in which it can be less painful, e.g.:

- move all the deprecated translations in core to a separate locale files (e.g. `solidus/core/config/locales/en.deprecated.yml`)
- copy the file to the host application as part of the solidus installation/upgrade process
- remove the deprecated file in the next major